apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -78,18 +102,52 @@ const AccessManagement = (): JSX.Element => {
 					query: { page: userPage, pageSize: userPageSize },
 				}),
 			);
-			onModalClose();
+			onCreateModalClose();
 		},
-		[dispatch, onModalClose, userPage, userPageSize],
+		[dispatch, onCreateModalClose, userPage, userPageSize],
 	);
 
+	const [groupToEdit, setGroupToEdit] =
+		useState<GroupGetAllItemResponseDto | null>(null);
+
+	const handleGroupUpdateSubmit = useCallback(
+		(id: number, payload: GroupUpdateRequestDto) => {
+			void dispatch(groupActions.update({ id, payload }));
+		},
+		[dispatch],
+	);
+
+	const onEdit = useCallback(
+		(group: GroupGetAllItemResponseDto): void => {
+			setGroupToEdit(group);
+			onUpdateModalOpen();
+		},
+		[onUpdateModalOpen],
+	);
+
+	useEffect(() => {
+		if (groupCreateStatus === DataStatus.FULFILLED) {
+			handleUsersChange();
+			onCreateModalClose();
+		}
+	}, [groupCreateStatus, handleUsersChange, onCreateModalClose]);
+
+	useEffect(() => {
+		if (groupUpdateStatus === DataStatus.FULFILLED) {
+			handleUsersChange();
+			onUpdateModalClose();
+			setGroupToEdit(null);
+		}
+	}, [groupUpdateStatus, handleUsersChange, onUpdateModalClose]);
```

Comment:
@liza-veis About it https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/128#discussion_r1752149527
Is it okay?